problems/removeVowels.js

```diff
@@ -6,8 +6,15 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
-
+function removeVowels(str) {
+    let newStr = ""
+    // str = str.toLowerCase();
+    for (let letter of str) {
+        if (letter.toLowerCase() !== "a" && letter.toLowerCase() !== "e" && letter.toLowerCase() !== "i" && letter.toLowerCase() !== "o" && letter.toLowerCase() !== "u") {
```

Comment:
instead of this long condition, you could of placed the vowels inside of a string or array and used the includes method to figure out if the current letter is a vowel or not